dsm_arnitel2045_175.xml.fdm_material

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Generic ABS profile. The data in this file may not be correct for your specific machine.
+-->
+<fdmmaterial xmlns="http://www.ultimaker.com/material">
+    <metadata>
+        <name>
+            <brand>DSM</brand>
+            <material>Elastomer</material>
```

Comment:
I wanted to be as generic as possible. The intend was to make a distinction between hard thermoplastics and rubber like thermoplastics/elastomers. 
How about "hard material" and "flex material" or "plastic" and "elastomer".

If we start calling it TPC or PA copolymer, people will say: huh, wha???   

Or should we even go so far as to change all the brand descriptions into "Generic plastic" and "Generic elastomer". Then I would leave DSM away and we could call it TPC or PA copolymer.